### DISCUSSION TITLE
Introduction: Ivana

### DISCUSSION THREAD

**Discussion by ibearthling** [life-itself/community#518] [Introductions]
Hi all. 

I recently came across the LIfe Itself... (it's a funny thingto say, isn't it? )

I live in the Tarn, 700 m altitude, in the middle of the forests, surrounded by lakes and rivers.  Being in daily contact with nature, it is easy to see what is happening, or at least - it is easier than it was when I was living in the cities.  And no, it's not all romantic, much of it is very hard work.  But what is important to me is that it is real and honest.  None of those virtual experiences ... and no manipulation.  It is what it is.  

Currently, I work online for a Canadian Non-profit, where I do various things - a bit of project management, quality assurance, coordination, etc.  

Other than that, I grow food, am learning how to cultivate my health, learning to compost and trying to imagine turning this baby around... 
![Screenshot 2023-05-09 at 07 31 45](https://user-images.githubusercontent.com/132978129/237002926-fe031a14-bc5d-4f17-8bb7-32852d710845.png)

**Comment by rufuspollock** [comment]
Thank-you for saying hello and sharing your commitments. A very warm welcome 🎉

> I recently came across the LIfe Itself... (it's a funny thingto say, isn't it? )

😄 we just say "Life Itself" and it's part of the joke e.g. "I'm part of Life Itself" - yes you are!

> To turn this big baby around... this is what we will need to do...

👍👍

The hard part of simple things here is how 😄 It's like becoming a concert pianist. when i was a child i used to listen to glenn gould and dreamed of playing bach like him. it seemed so easy listening to him. ... and in theory it's just putting your fingers in this place on the piano keys at this time. But ... it's hard.

And that's why it is is so great to be in communication with you and others like you passionate to make a difference **and actively seeking to live that difference and take practical action to realize that more broadly** Because it will take a movement, or [specifically](https://lifeitself.org/tao/scqh#hypothesis):

> a federation of resilient, flourishing communities based on a shared culture [1] and engaged with broader society and allying with like-minded groups to foster a movement of wisdom and awakening and committed to practical action, including large scale political efforts to address collective challenges and transform wider society.

[1]: views, values and practices including the possibility of transformation, the primacy of (inter)being and getting stuff done

**Reply by rufuspollock** [reply]
@ibearthling 👋 and checking you saw this. Thanks for showing up here!

**Reply by ibearthling** [reply]
Hi Rufus.. yeah.. I saw that - thank you for responding.   I saw nothing other than that, though, which I assumed probably meant that what I said didn't resonate strongly enough for there to be more reactions.  I visited the page daily for some time, tried to see what was happening and what i could get involved in...  But then it all got kind of quiet, so I didn't visit any more.  And my ideas also evolved somewhat so when I had the time for it I tried to tease them out onto the paper... LIke many people, I don't have much free time, and like many people, I know that's a kind of lame excuse... But still... 
I hope things are well in your community... I'll go and look around now, see what interesting new things appeared since my last visit.

**Reply by nathenf** [reply]
Hi Ivana, @ibearthling 

Welcome to the community! I don't know which part of the community you are exploring but we are actually most active on our Whatsapp Chat which you can join here: https://chat.whatsapp.com/JNJCTZugNQn1fq89xbHtfA if you haven't joined already. We also have bi-monthly community calls and the next one is on Thursday at 7:30pm CET https://lifeitself.org/blog/2023/08/01/The-art-of-creating-intentional-futures-Jan-Demiralp if you would like to join 😄
